src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -93,6 +93,9 @@ public void storeReply(Message reply) {
     replyEntity.setProperty("recipient", reply.getRecipient());
     replyEntity.setProperty("sentimentScore", reply.getSentimentScore());
     replyEntity.setProperty("timestamp", reply.getTimestamp());
+    if(reply.getImageUrl() != null) {
+      replyEntity.setProperty("imageURL", reply.getImageUrl());
+    }
```

Comment:
So replies can have images uploaded? Is there a way for users to actually do that?

Reply:
Hmm I'll have to look at Gloria's code, but I could probably make it happen!